Elevel/Elevel.Api/Controllers/UserController.cs

```diff
@@ -25,5 +25,20 @@ public async Task<IActionResult> GetTokenAsync(TokenRequestModel model)
             var result = await _userService.GetTokenAsync(model);
             return Ok(result);
         }
+
+        [HttpGet("info")]
+        public async Task<IActionResult> UserInfo()
+        {
+            var claims = User.Claims.ToList();
+            var userId = claims.FirstOrDefault(x=>x.Type == "uid").Value;
+            GetApplicationUserByIdQuery.Request request = new GetApplicationUserByIdQuery.Request()
+            {
+                Id = Guid.Parse(userId)
```

Comment:
this can be failed too, it string will contain not valid guid value or invalid format